src/jp/mcbe/fuyutsuki/Texter/Main.php

```diff
@@ -70,12 +70,14 @@ public function onLoad(): void {
 
 	public function onEnable(): void {
 		$pluginManager = $this->getServer()->getPluginManager();
-		if ($this->isPackaged()) {
```

Comment:
Why disable the PHAR check?

Reply:
whoops, forgot to enable it again after testing